Formula/macvim.rb

```diff
@@ -56,7 +55,8 @@ def install
                           "--with-lua-prefix=#{Formula["lua"].opt_prefix}",
                           "--enable-luainterp",
                           "--enable-python3interp",
-                          "--disable-sparkle"
+                          "--disable-sparkle",
+                          "--with-macarchs=#{arch}" # remove when universal binaries are supported
```

Comment:
I don't think this comment `# remove when universal binaries are supported` is correct.

Homebrew, AFAIK, has no plans to support fat binaries. That means MacVim would not be able to build universal binaries in Homebrew as well, since it depends on `gettext` to work. It will always build thin binaries at least under the current design.

Reply:
```suggestion
                          "--with-macarchs=#{arch}"
```

Just remove the comment then?

Reply:
Yeah that’s my vote. Just remove it to avoid confusion.